freeipa.spec.in

```diff
@@ -248,6 +248,7 @@ Requires: %{name}-server-common = %{version}-%{release}
 Requires: %{name}-common = %{version}-%{release}
 Requires: python2-ipaclient = %{version}-%{release}
 Requires: python-ldap >= 2.4.15
+Requires: python-lxml
```

Comment:
file `odsmgr.py` is in  python2-ipalib, so dependency python-lxml should stay with python2-ipalib not ipaserver
```
$ rpm -ql python2-ipalib | grep odsmgr
/usr/lib/python2.7/site-packages/ipapython/dnssec/odsmgr.py
/usr/lib/python2.7/site-packages/ipapython/dnssec/odsmgr.pyc
/usr/lib/python2.7/site-packages/ipapython/dnssec/odsmgr.pyo

$ # rpm -ql python2-ipaserver | grep odsmgr
$
```

You can move the whole dnssec dir/package into ipaserver with all related dependencies (in new patch) or keep python-lxml dependency with python-ipalib